packages/oruga-next/src/components/dropdown/Dropdown.vue

```diff
@@ -500,6 +512,19 @@ const rootClasses = defineClasses(
         null,
         computed(() => isMobileModal.value && !hoverable.value),
     ],
+    [
+        "rootPositionClass",
+        "o-drop--position-",
+        autoPosition,
+        computed(() => !!autoPosition.value),
+    ],
+    [
+        "rootActiveClass",
```

Comment:
I think it's fine that the class is just called `activeClass' and `positionClass' without the `root` like the other.

Reply:
Fixed.